### ISSUE TITLE
[FEATURE REQUEST] Add download all from hoster

### ISSUE THREAD

**Issue by Invictaz** [LivingWithHippos/unchained-android#160]
Add download all from hoster. Relaxed to multiple select from hoster.

**Comment by LivingWithHippos** [comment]
Mmmh these are all referring to the same file from different links, you don't want to download all of them right?

**Comment by Invictaz** [comment]
No, each file you see in the screenshot is a part of the 720p episode

**Comment by LivingWithHippos** [comment]
No I don't think so ;D maybe you sent the wrong screenshot?

**Comment by Invictaz** [comment]
All links are from Rapidgator. And with the same filename. But the filenames are truncated. So the part numbers fell off.

**Comment by LivingWithHippos** [comment]
No, you have multiple versions because rlsbb let people post new links in the comments. If you go  [to the page here](https://link.infini.fr/24uJp0PX) you can see that there are no parts

**Comment by Invictaz** [comment]
The issue still stands, this was just a bad example. There are files from hosters which are split into multiple parts. Which cannot be downloaded now.

**Comment by LivingWithHippos** [comment]
Rlsbb has lot of links and most of them are from the comments. Some pages are 60+ links, it's not possible to add a button that downloads only the stuff you want, but it could still be added and used if you are sure of what you're doing
